Include epoch in attestation/sync committee performance logs (#4538)
PegaSysEng/artemis: CHANGELOG.md

- Support v.2.1.0 of the standard rest api. it should be noted that the 'version' has been changed to lower case to comply with the api specification.
- Attestations are now sent to the beacon node in batches by default when using the validator-client.
- Updated to Javalin 4 for the rest api.

> - Included epoch in attestation and sync committee performance log message.

Reviewer: ```suggestion
- Added relevant epoch to attestation and sync committee performance log message.
```
